packages/eslint-plugin/docs/rules/prefer-at.md

```diff
@@ -0,0 +1,101 @@
+---
+description: 'Enforce the use of `array.at(-1)` instead of `array[array.length - 1]`.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/prefer-at** for documentation.
+
+There are two ways to get the last item of the array:
+
+- `arr[arr.length - 1]`: getting an item by index calculated relative to the length of the array.
+- `arr.at(-1)`: getting an item by the `at` method. [Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)
```

Comment:
[Proofreading] Informative link titles makes it easier to navigate through the page with a screenreader. And this is a bit less text to read:

```suggestion
- `arr.at(-1)`: getting an item by the [`at` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at).
```